sdk/core/core-http/lib/coreHttp.ts

```diff
@@ -13,7 +13,7 @@ export { OperationArguments } from "./operationArguments";
 export { OperationParameter, OperationQueryParameter, OperationURLParameter } from "./operationParameter";
 export { OperationResponse } from "./operationResponse";
 export { OperationSpec } from "./operationSpec";
-export { ServiceClient, ServiceClientOptions, flattenResponse } from "./serviceClient";
+export { ServiceClient, ServiceClientOptions, flattenResponse, ProxySettings } from "./serviceClient";
```

Comment:
This is the only change related to core-http in this PR.

Made a separate PR to master - https://github.com/Azure/azure-sdk-for-js/pull/5043.

I will then merge the master branch into feature/storage and the merge this PR(basic auth support for storage-blob).